packages/usa-table/src/usa-table.stories.js

```diff
@@ -20,6 +20,11 @@ export default {
       control: { type: "boolean" },
       defaultValue: false,
     },
+    table_footer: {
+      name: "Include table footer",
+      control: { type: "boolean" },
+      defaultValue: false,
+    },
```

Comment:
**issue**: Can we remove this and related code for now since we only have hardcoded values?